jira_sync/__init__.py

```diff
@@ -1,2 +1 @@
-"""Init file for jira_sync module."""
-__version__ = "0.1.0"
+from .version import __version__
```

Comment:
I think this was probably generated by `poetry init`

Reply:
Could be – I threw it in, then we can simply `poetry version ...` to bump it and not have to remember to do it elsewhere, too.

Reply:
That is always better